modules/imgproc/test/test_houghlines.cpp

```diff
@@ -140,6 +140,29 @@ class ProbabilisticHoughLinesTest : public BaseHoughLineTest, public testing::Te
     }
 };
 
+typedef std::tr1::tuple<double, double, double, double> HoughLinesPointSetInput_t;
```

Comment:
Don't use `std::tr1` here and below: https://github.com/opencv/opencv/wiki/Coding_Style_Guide#implementing-tests